lib/api/helpers.js

```diff
@@ -1,5 +1,5 @@
 import url from "url";
-import { slugify } from "transliteration";
+import "transliteration/lib/browser/transliteration.js";
```

Comment:
I think we might need to do something like this (this is what was there in a previous incarnation)
```
if (Meteor.isServer) {
  import { slugify } from "transliteration";
}

```